rdkit/Chem/Pharm2D/UnitTestMatcher.py

```diff
@@ -8,14 +8,17 @@
 #  which is included in the file license.txt, found at the root
 #  of the RDKit source tree.
 #
+from rdkit.TestRunner import redirect_stdout
```

Comment:
These should not be over the doctstring for the file.
Given that this particular docstring (the one starting on l13) is useless, it makes sense to just remove that.